js/templates/modals/listingDetail/listing.html

```diff
@@ -48,7 +59,7 @@ <h2 class="txUnb"><%= ob.convertAndFormatCurrency(ob.item.price, ob.metadata.pri
                 </div>
               </div>
               -->
-              <button class="btnHg clrBAttGrad clrBr3 clrTOnEmph">Buy Now</button>
+              <button class="btnHg clrBAttGrad clrBr3 txUp clrTOnEmph">Buy Now</button>
```

Comment:
I think it would be better to capitalize this in the language file, since we don't know if capitalization would be appropriate for all languages.